src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -2036,6 +2045,7 @@ SubquerySqlTaskList(Job *job)
 		uint32 tableId = rangeTableIndex + 1; /* tableId starts from 1 */
 		uint32 finalShardCount = 0;
 		uint32 shardIndex = 0;
+		uint64 tableSize = 0;
```

Comment:
As far as I understand, we are not calculating the whole `tableSize` here. Should we re-name this variable to reflect that we're only calculating the size of shards that the query hits?